flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStStateExecutor.java

```diff
@@ -73,6 +74,11 @@ public class ForStStateExecutor implements StateExecutor {
     /** The ongoing sub-processes count. */
     private final AtomicLong ongoing;
 
+    /**
+     * Whether the executor has been closed, if closed, the new state requests would be rejected.
+     */
+    private boolean closed;
```

Comment:
I thought this is not necessary since the `executor.shutdown()` will reject the future tasks.

Reply:
Removed.